core/src/main/java/com/microsoft/applicationinsights/telemetry/EventTelemetry.java

```diff
@@ -37,13 +37,13 @@
     /**
      * Envelope Name for this telemetry.
      */
-    private static final String ENVELOPE_NAME = "Event";
+    public static final String ENVELOPE_NAME = "Event";
```

Comment:
Are you accessing all these directly in LF channel or somewhere? If so I would say that this can introduce more troubles. What if the user creates a Telemetry of this type and unknowing change the envelope name? It can cause more tickets. I would suggest a careful revisit of these.

Reply:
No, I was using this as the key to the transformer map, but I changed my mind to use BASE_TYPE, so these can be reverted. On it.

Reply:
(Deleting my previous comment)
I actually do need `MetricTelemetry.ENVELOPE_NAME` and it didn't make sense to me to just expose one. Tests depend on it right now.

Reply:
I am a bit lost .... ;) If the test is the only dependency of this, I would say rather not expose this and use reflection in tests (I know reflection is not ideal but it is an apt tool for this scenario). Exposing this is hazardous IMO.

Reply:
I reverted the change because the tests were failing and I wanted to see if everything else was passing (it is). 

I looked for other options (to not expose new fields/methods) and this seemed like the best choice. We should meet with the team on Tuesday.

Reply:
Unfortunately not. The LocalForwarderChannel is located in a different package. It wouldn't make sense for either to be relocated to the same package.

The other option I thought of is to create a third class which is public and has all these strings which we need for this and LocalForwarder. I'm happy to refactor if it's an issue to expose these constants.